src/stories/containers/transparency-report/transparency-comments/comments-list.tsx

```diff
@@ -13,6 +14,8 @@ interface Props {
 export const ListItemsComments = ({ comments, code }: Props) => {
   const { isLight } = useThemeContext();
 
+  return <AuditorCommentsContainer />;
+
   return (
```

Comment:
Unreachable code here

Reply:
This section of the component is now unreachable because the new version of the comments is not completed, this PR only contains the layout of the new feature, after the integration with the API is done this component will be modified/deleted

Reply:
Ok then, good to me in that case